segmentstore/server/src/main/java/io/pravega/segmentstore/server/store/ServiceConfig.java

```diff
@@ -58,6 +58,8 @@
     public static final Property<String> REST_LISTENING_HOST = Property.named("rest.listener.host", "localhost");
     public static final Property<Integer> REST_LISTENING_PORT = Property.named("rest.listener.port", 6061);
     public static final Property<Boolean> REST_LISTENING_ENABLE = Property.named("rest.listener.enable", true);
+    public static final Property<String> REST_KEYSTORE_FILE = Property.named("security.tls.server.keyStore.location", "", "rest.tlsKeyStoreFile");
```

Comment:
You don't need the third argument to `Property.named` here as this is not a setting that has been upgraded from 0.8 (when we renamed all these settings). Let's remove the `rest.tlsKeyStoreFile` (this line) and `rest.tlsKeyStorePasswordFile` (next line).

Reply:
Fixed, removed the legacy names.